inference/server/oasst_inference_server/models/worker.py

```diff
@@ -17,7 +18,7 @@ class WorkerEventType(str, enum.Enum):
 class DbWorkerComplianceCheck(SQLModel, table=True):
     __tablename__ = "worker_compliance_check"
 
-    id: str = Field(default_factory=lambda: str(uuid4()), primary_key=True)
+    id: str = Field(default_factory=lambda: uuid7str(), primary_key=True)
```

Comment:
is the lambda necessary or could it be `default_factory=uuid7str` ?

Reply:
Thanks, I'm not really familiar with python syntax.